web-client/public/js/upload.js

```diff
@@ -252,28 +276,38 @@ $(function () {
     var emptyRowCount = warnings.filter(function(x){return x.warningCode=="EMPTY_ROW"});
     var invalidNetworkSizeCount = warnings.filter(function(x){return x.warningCode=="INVALID_NETWORK_SIZE"});
     var extraneousDataCount = warnings.filter(function(x){return x.warningCode=="EXTRANEOUS_DATA"});
+    var edgesWithoutWeightsCount = warnings.filter(function(x){return x.warningCode=="EDGES_WITHOUT_WEIGHTS"});
+    var edgeDefaultNotDirectedCount = warnings.filter(function(x){return x.warningCode=="EDGE_DEFAULT_NOT_DIRECTED"});
+    var sifFormatWarningCount = warnings.filter(function(x){return x.warningCode=="SIF_FORMAT_WARNING"});
 
```

Comment:
Oh, I meant that everything was very tight, unlike the rest of the code. Plus I am just now noticing that `==` is being used and not `===`.

I would have formatted the code as follows:

    var edgeDefaultNotDirectedCount = warnings.filter(function (x) { return x.warningCode === "EDGE_DEFAULT_NOT_DIRECTED"; });